### PR TITLE
Speed up GFA import, add chr22.gfa benchmark file, fix GFA export formatting issues

### PR DESCRIPTION
**Pull request by dkhofer**

### CODE REVIEW

File: src/exports/gfa.rs [hunk 1]
```diff
@@ -181,9 +185,14 @@ fn write_paths(
             }
         }
 
+        let full_path_name = if sample_name.is_some() {
```
Comment (Chris7) [inline src/exports/gfa.rs:188]: i'm kind of surprised the linter let you do this, it always yells at me to do `if let Some(full_path_name) = sample_name``

File: src/exports/gfa.rs [hunk 2]
```diff
@@ -181,9 +185,14 @@ fn write_paths(
...
+            format!("{}.{}", path.name, sample_name.unwrap()).to_string()
```
Comment (Chris7) [inline src/exports/gfa.rs:189]: can we check if sample_name is actually the empty string so we don't prefix the reference chromosomes w/ `.`

**Review by Chris7 — APPROVED** [review]
LGTM. One issue on the path name convention to handle the null sample case.

**Comment by Chris7** [comment]
oh, can you sneak the strand::forwrd change in here too for boundary edges?

**Comment by dkhofer** [comment]
> oh, can you sneak the strand::forwrd change in here too for boundary edges?

Yep, snuck

**Pull request merged**